docs/windows.md

```diff
@@ -313,12 +313,16 @@ Set-NetFirewallProfile -Profile Domain,Public,Private -Enabled False
 Firstly, install wins, kubelet, kubeadm using script `PrepareNode.ps1` provided
 by kubernetes. The third component [`wins`](https://github.com/rancher/wins) is
 used to run kube-proxy and antrea-agent on Windows host inside the Windows
-container.
+container. Specify the Node IP , Kubernetes Version and Container Runtime while 
+running the script. If you do not specify any container runtime , it will be Docker 
+by default. (Note:Docker has been deprecated since k8s version 1.24).You can also 
+specify whether you want to install OVS on the node.
 
 ```powershell
 # Example:
 curl.exe -LO "https://github.com/kubernetes-sigs/sig-windows-tools/releases/download/v0.1.5/PrepareNode.ps1"
-.\PrepareNode.ps1 -KubernetesVersion v1.23.5
+.\Prepare-Node.ps1 -KubernetesVersion v1.27.x -InstallOVS -NodeIP 192.168.1.10 -ContainerRuntime ContainerD
```

Comment:
"ContainerD" is a weird writing, can we just use "docker" and "containerd" in this option?

Reply:
Use a valid K8s version in the example rather than "1.27.x"?